src/components/FiltersDrawer/FiltersDrawer.tsx

```diff
@@ -101,8 +127,29 @@ export const FiltersDrawer = ({
     }
   }, [filtersBody]);
 
-  const buttonVariant = hasSelectedFilters ? 'primary' : 'secondary';
-  const buttonStatus = hasSelectedFilters ? 'brand' : 'neutral';
+  const addFiltersToTriggerElement = (trigger: ReactNode) => {
+    if (!React.isValidElement(trigger)) return;
```

Comment:
I put this line in for accessing `props`/`cloneElement` if this helper function is copypasta'd elsewherem but not exactly necessary here since `trigger` is already being typechecked as ReactNode and truthy checked in the ternary in line 143. 

Let me know if I should remove

Reply:
Seems fine to me 🤷